lib/azure/armrest/model/storage_account.rb

```diff
@@ -64,27 +64,55 @@ def table_info(table, key = nil)
       # account +key+. The exact nature of the TableData object depends on the
       # type of table that it is.
       #
+      # You may specify :filter, :select or :top as options to restrict your
+      # result set.
+      #
+      # By default you will receive a maximum of 1000 records. If you wish to
+      # receive more records, you will need to use the continuation token. You
+      # may also set the :all option to true if you want all records, though we
+      # recommend using a filter as well if you use that option as there can
+      # be thousands of results.
+      #
+      # You may also specify a :NextRowKey, :NextPartitionKey or :NextTableset
+      # explicitly for paging. Normally you would just pass the
+      # collection's continuation_token, however. See below for an example.
+      #
+      # When using continuation tokens, you should retain your original
+      # filtering as well, or you may get unexpected results.
+      #
+      # Examples:
+      #
+      #   # Get the first 10 rows of data from the last 3 days
+      #   date = (Time.now - (86400 * 3)).iso8601
+      #   my_filter = "timestamp ge datetime'#{date}'"
+      #   options = {:top => 10, :filter => my_filter}
+      #
+      #   results = storage_account.table_data(table, key, options)
+      #
+      #   # Now get the next 10 records
+      #   if results.continuation_token
+      #     options.merge!(results.continuation_token)
```

Comment:
No, because for storage accounts, the continuation_token is actually a hash of up to 3 key/value pairs.